lib/src/tmx_object.dart

```diff
@@ -4,11 +4,11 @@ class TmxObject {
   String name;
   String type;
 
-  int x;
-  int y;
-  int width = 0;
-  int height = 0;
-  int rotation = 0;
+  double x;
```

Comment:
I changed the type to `double`. But this introduces a breaking change I suppose?

Reply:
it does, let's release it under 0.6.0. I can update flame shortly after, I think we are doing 0.24 soon-ish :)